README.md

```diff
@@ -7,3 +7,25 @@ Item Manufacturing and Sales
 
 # Purpose
 This is a project for CSU Fresno CSCI 226. The concept is to show how relational databases work. The project is going to use a real world example of how items are built from raw materials and then selling those items to customers.
+
+# Trigger
```

Comment:
Why would you do this?

Reply:
The professor is requiring a trigger so I thought this was the easiest and something I would want done anyways.

Reply:
Bad professor.

Reply:
If you must include it I would at least make sure it is up/down'd by a migration.